questionnaire/questionnaire/utils/taskRunner/tasks/transformAndUpload/index.test.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const {transformQuestionnaire} = require('.');
+const questionnaireFixture = require('../../../../../test-fixtures/res/questionnaireCompleteForCheckYourAnswers');
+const questionnaire = require('../../../../questionnaire');
+
+describe('Integration Service', () => {
+    // Arrange
+    const questionnaireObj = questionnaire({questionnaireDefinition: questionnaireFixture});
+
+    // Act
+    const result = transformQuestionnaire(questionnaireObj);
```

Comment:
This are being shared be each test, meaning the tests aren't independent of each other. Can you use an appropriate setup/teardown or have each test do these steps within their own function body